README.Rmd

```diff
@@ -1,20 +1,8 @@
 ---
-output: 
-  github_document:
-    toc: false
-    fig_width: 10.08
-    fig_height: 6
-tags: [r, effect size, standardized]
-vignette: >
-  %\VignetteIndexEntry{README}
-  \usepackage[utf8]{inputenc}
-  %\VignetteEngine{knitr::rmarkdown}
-csl: vignettes/apa.csl
-editor_options: 
-  chunk_output_type: console
```

Comment:
None of this is needed or relevant for github-flavored Markdown document. 

Also, we don't set these options for any of the other repos, so there is also an argument for consistency.